packages/instructor-stream/src/utils/tokenizer.ts

```
@@ -152,490 +150,487 @@
+      // Signal error and stop processing
+      return this.error(
+        new TokenizerError(
           `Unexpected "${String.fromCharCode(
             n
           )}" at position "${i}" in state ${TokenizerStateToString(this.state)}`
         )
-      }
-      // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    } catch (err: any) {
-      this.error(err)
+      )
```

Biome (2.1.2)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)